code/game/machinery/computer/robot.dm

```diff
@@ -142,3 +162,42 @@
 					drone.visible_message(span_danger("\the [drone] self-destructs!"))
 					drone.investigate_log("has been gibbed by a robotics console.", INVESTIGATE_DEATHS)
 					drone.gib()
+
+
+// I feel like this should be changed, but I have no idea in what way exactly, so I just extracted it to make the code less of a mess
+/obj/machinery/computer/robotics/proc/lock_unlock_borg(mob/living/silicon/robot/R, console_location = null)
```

Comment:
I'm not going to make you change use of single-letter variable R across the entire file but could you at least do it for this proc (and any other new one you make)?

Reply:
This way its bad, but at least its consistent across the file.